app/assets/stylesheets/custom.scss

```diff
@@ -321,3 +321,25 @@ ul, menu, dir {
     width: 100px;
     height: 30px;
 }
+
+#flash_notice {
+  &.error {
+    border: solid 4px #900;
+          background-color: #FF6565;
+        }
+  &.notice {
+    border: solid 4px #26722D;
```

Comment:
Color `#26722D` should be written as `#26722d`<br>Properties should be ordered background-color, border